edenai_apis/apis/sentisight/sentisight_api.py

```
@@ -160,49 +160,66 @@ def image__object_detection(
     async def image__aobject_detection(
         self, file: str, file_url: str = "", model: Optional[str] = None, **kwargs
```

Ruff (0.14.5)

Unused method argument: `model`

(ARG002)